apps/docs/docs/state/concepts-and-best-practices/concepts-and-best-practices.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 5
+sidebar_position: 70
```

Comment:
Why are you changing the sidebar position to x10  number? 
Did you mean 7?

Reply:
This is the 4th item from the end.

Reply:
I'm using a factor of 10 to make room for additional items as I don't like floating-point numbers as indices.